web/styles/popovers.css

```diff
@@ -917,12 +915,17 @@ ul.popover-group-menu-member-list {
     }
 
     .move_messages_edit_topic {
-        margin-bottom: 10px;
+        margin-bottom: unset;
+        width: 97%;
```

Comment:
From the `width: 97%`, that I see over here, seems like setting the `width` to `100%` is causing the input box to overflow? Did you happen to find a reason for that?

Reply:
Maybe try setting `box-sizing` to `border-box` on the input element, and then adjusting the padding? I think you'll also need remove/modify the `height` property on the input element as well.

Reply:
Yes, setting the `width` to `100%` caused the input box to overflow. I tried to identify the cause but was unsure if the behavior was related to the `simplebar` classes. I will keep this in mind and will put in more effort to debug if similar issues arise in the future.

Also, setting `box-sizing` to `border-box` works well. Thank you for the valuable feedback.